platform/MCU/nRF52840/inc/dct.h

```diff
@@ -107,9 +108,9 @@ typedef struct __attribute__((packed)) application_dct {
     uint8_t device_secret[15];           // Device secret data (aka "mobile secret")
     uint8_t setup_done;                  // Setup done flag
     uint32_t ncp_id;                     // NCP identifier
-    uint8_t pm_detect;                   // Detect power management peripherals flag
+    hal_power_config power_config;       // Power management configuration
     uint8_t radio_antenna;               // Mesh/BLE antenna: 0x01 - internal, 0x02 - external, 0xff - default
```

Comment:
The power_config broke the radio_antenna setting. `selectAntenna(radio_antenna_type antenna)`  doesn't handle the default flash value 0xFF, so the GPIO of antenna switch will be left in the default mode.

Reply:
This is deliberate. This feature (antenna setting) has not been released in any public DeviceOS release, so we can still make such changes.

Reply:
I see, we have the default setting `RADIO_ANT_DEFAULT` in `initRadioAntenna()`, mark it resolved.